metabase/metabase
# Make search icon bigger in the question filter

Closes https://github.com/metabase/metabase/issues/31814

### Description

Make search icon bigger when input is not shown

### How to verify

- Start a new question on any table
- Visualise it
- Click the "Filter" button (top right)
- Filter modal is displayed and you'll see that the magnifying glass icon is not small (top right beside the close "x")

### Demo

https://www.loom.com/share/0e72fe12f85c4a378e9b6bb9f6c041a4

### Checklist

~- [ ] Tests have been added/updated to cover changes in this PR~
